colour/models/rgb/transfer_functions/__init__.py

```diff
@@ -58,6 +58,7 @@
                         log_encoding_SLog3, log_decoding_SLog3)
 from .srgb import eotf_inverse_sRGB, eotf_sRGB
 from .viper_log import log_encoding_ViperLog, log_decoding_ViperLog
+from .log import log_encoding_Log2, log_decoding_Log2
```

Comment:
Given the module does not depend on anything nor anything really depends on it, please import it in alphabetical order.

Reply:
Noted! Will take care of that in the future! :+1: